examples/nuxt-3/components/ProductCard.vue

```diff
@@ -0,0 +1,150 @@
+<template>
+  <div class="product-card">
+    <div class="product-card__main">
+      <h2
+        class="product-card__title"
+        :id="`product-card-header-${product.content.handle}`"
+      >
+        <NuxtLink :to="`/products/${product.content.handle}`">{{
+          product.content.title
+        }}</NuxtLink>
+      </h2>
+      <div v-if="selectedVariant" class="product-card__prices">
+        <del
+          v-if="selectedVariant.compareAtPrice"
+          class="product-card__compare"
+        >
+          ${{ selectedVariant.compareAtPrice }}
+        </del>
+        <div class="product-card__price">${{ selectedVariant.price }}</div>
+      </div>
+      <div
+        class="product-card__option"
+        v-for="option in options"
+        :key="option.name"
+      >
+        <label
+          :for="`${option.name}-select-id`"
+          class="product-card__option-label"
+          >{{ option.name }}</label
+        >
+        <select :id="`${option.name}-select-id`" class="product-card__select">
+          <option
+            v-for="(value, vIndex) in option.values"
+            :value="value"
+            :key="value"
+          >
+            {{ value }}
+          </option>
```

Comment:
<img width="600" alt="vue:no-unused-vars  'vIndex' is defined but never used" src="https://user-images.githubusercontent.com/5732000/185222933-fc4acda9-dfcb-45d0-82ac-f4c943af451d.png">

```suggestion
          <option v-for="value in option.values" :value="value" :key="value">
            {{ value }}
          </option>
```